### ISSUE TITLE
简单版demo 报错了

### ISSUE THREAD

**Issue by evelynEdison** [zhongjhATC/AlbumCameraRecorder#54]
java.lang.RuntimeException: Unable to destroy activity {com.zhongjh.cameraapp/com.zhongjh.albumcamerarecorder.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.google.android.material.tabs.TabLayoutMediator.detach()' on a null object reference
        at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:5309)
        at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:5339)
        at android.app.servertransaction.DestroyActivityItem.execute(DestroyActivityItem.java:44)
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2246)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:233)
        at android.app.ActivityThread.main(ActivityThread.java:8010)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:631)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:978)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.google.android.material.tabs.TabLayoutMediator.detach()' on a null object reference
        at com.zhongjh.albumcamerarecorder.MainActivity.onDestroy(MainActivity.java:138)
        at android.app.Activity.performDestroy(Activity.java:8251)
        at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1364)
        at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:5294)
        at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:5339) 
        at android.app.servertransaction.DestroyActivityItem.execute(DestroyActivityItem.java:44) 
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2246) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:233) 
        at android.app.ActivityThread.main(ActivityThread.java:8010) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:631) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:978)

**Comment by zhongjhATC** [comment]
请根据 https://github.com/zhongjhATC/AlbumCameraRecorder/issues/46 规范提问谢谢合作

**Comment by evelynEdison** [comment]
好的，已重新发布
